tests/test_ses/test_ses_boto3.py

```diff
@@ -132,6 +132,21 @@ def test_send_templated_email():
     conn.send_templated_email.when.called_with(**kwargs).should.throw(ClientError)
 
     conn.verify_domain_identity(Domain="example.com")
+
+    with pytest.raises(ClientError) as ex:
+        conn.send_templated_email(**kwargs).when.called_with(**kwargs)
```

Comment:
The `.when.called_with(**kwargs)` part is not necessary here

Reply:
@bblommers Oops. Fixed 👍🏼